web/stories/ui/formatted-text.stories.svelte

```diff
@@ -0,0 +1,227 @@
+<script context="module" lang="ts">
+  import imgExample2 from '../../public/stories-assets/images/long_width_example.avif'
+  import imgExample1 from '../../public/stories-assets/images/short_width_example.png'
+  import FormattedText from '../../ui/formatted-text.svelte'
+
+  export const meta = {
+    component: FormattedText,
+    title: 'UI/FormattedText'
+  }
+</script>
+
+<script lang="ts">
+  import { Story } from '@storybook/addon-svelte-csf'
+
+  import Card from '../../ui/card.svelte'
+  import Section from '../section.svelte'
+
+  let fishText =
+    'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut culpa cumque, dolorum nobis quidem repellendus sunt! Amet assumenda aut deserunt, nest facilis nihil officiis pariatur quia quisquam sapiente tempora vitae.'
+
+  let paragraph = `<p>${fishText}</p>`
+
+  let longHeadingText =
+    'This is a very long heading text that goes on and on. It is meant to test the rendering of long headings in the UI. It includes many words and phrases to ensure that the heading wraps correctly and maintains its styling.'
+
+  let longHeadings = `<h1>${longHeadingText}</h1><h2><a href="#">${longHeadingText}</a></h2><h3>${longHeadingText}</h3><h5>${longHeadingText}</h5><h6>${longHeadingText}</h6>`
+
+  let headingsWithParagraphs = `<h1>Heading H1</h1>${paragraph}${paragraph}<h2>Heading H2</h2><p>${paragraph}${paragraph}${paragraph}</p><h3>Heading H3</h3><p>${paragraph}</p><h4>Heading H4</h4><p>${paragraph}</p><h5>Heading H5</h5><p>${paragraph}</p><h6>Heading H6</h6><p>${paragraph}</p>`
+
+  let images =
+    `<img src="${imgExample1}" alt="" loading="lazy">` +
+    `<img src="${imgExample2}" alt="">`
+
+  let inlineTagsExample =
+    '  <p><b>Bold</b> Text Example</p>\n' +
+    '  <p><i>Introduction</i></p>\n' +
+    '  <p><strong>Fish</strong> are a group of <em>aquatic</em> animals that first appeared during the <mark>Devonian</mark> period.</p>\n' +
+    '  <p><bdi>Types of Fish</bdi></p>\n' +
+    '  <p><bdo>Characteristics of Fish</bdo></p>\n' +
+    '  <p>Fish are characterized by their <strong>gills</strong>, <em>fins</em>, and <mark>scales</mark>.</p>\n' +
+    '  <p><cite>Interesting Facts</cite></p>\n' +
+    '  <p>Did you know that the <data value="whale-shark">whale shark</data> is the largest fish in the world?</p>\n' +
+    '  <p><del>Conclusion</del></p>\n' +
+    '  <p><ins>Fish are fascinating creatures!</ins> They come in all shapes and sizes, from the tiny <small>paedocypris</small> to the massive <samp>whale shark</samp>.</p>\n' +
+    '  <p><sub>Subscript text</sub> and <sup>superscript text</sup> can also be used for scientific notation.</p>\n' +
+    '  <p>For more information, visit our <a href="https://example.com">website</a>.</p>\n'
+
+  let unorderedList =
+    '<ul>\n' +
+    '  <li>Simple list item</li>\n' +
+    '  <li><a href="https://example.com">Item with link</a></li>\n' +
```

Comment:
We need a visual test for new and visited link.

I suggest:

1. For new link use random domain `example + random + .com`
2. For visited link we can use `href=""` to the current page